spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -74,7 +74,12 @@ An alternative is selected for the bean archive if either:
 ----
 
 Each child `<class>` element must specify the name of a bean class of an alternative bean.
```

Comment:
If we go for it (I think it's basically ok but on the other hand we are going to confuse the users again little bit) then this sentence needs some change as well IMHO. Because when I veto it then it's not a bean anymore. So just something like?:
` Each child `<class>` element must specify the name of a class of an alternative bean candidate.` or 
`Each child `<class>` element must specify the name of a class which is annotated with @Alternative.`
WDYT?